hbase-protocol-shaded/src/main/protobuf/server/master/Master.proto

```diff
@@ -696,6 +696,7 @@ message ListDecommissionedRegionServersResponse {
 message DecommissionRegionServersRequest {
   repeated ServerName server_name = 1;
   required bool offload = 2;
+  optional bool match_host_name_only = 3 [default = false];
```

Comment:
Should we also support a `match_hostname_and_port_only` variation on this feature?